### PR TITLE
Adding symbol to the WorkflowMultiBranchProject class descriptor

### PR DESCRIPTION
**Pull request by joseblas**
The WorkflowMultiBranchProject class does not have a proper Symbol in the descriptor as many other such as FreeStyleProject (https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/model/FreeStyleProject.java#L79)

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java [hunk 1]
```diff
@@ -133,7 +134,8 @@ private BranchJobProperty reconstructBranchJobProperty(@NonNull WorkflowJob job,
                 strategy != null ? strategy.getPropertiesFor(head) : Collections.emptyList()));
     }
 
-    @Extension public static class DescriptorImpl extends MultiBranchProjectDescriptor implements IconSpec {
+    @Extension(ordinal = 1000) @Symbol({"multibranch"})
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java:137]: Why the `ordinal`? Is that somehow related to adding a symbol?

File: src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java [hunk 2]
```diff
@@ -267,4 +271,16 @@ public DescriptorImpl() {
         r.assertLogContains("branch=feature2", b2);
     }
 
+    @Test public void getProjectClassBySymbol(){
```
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java:274]: This is not really testing anything not apparent from source code. Unless there is some user-visible effect of the change that can be asserted, may as well just skip the test coverage.

**Review by jglick — COMMENTED** [review]
Note that the display name currently specifies _Pipeline_ https://github.com/jenkinsci/workflow-multibranch-plugin/blob/e4aa6a85ba848d6759c0a46493d710e74e3272b6/src/main/resources/org/jenkinsci/plugins/workflow/multibranch/Messages.properties#L2 OTOH there is no other seriously maintained multibranch impl (`literate` plugin is long dead, as is `yaml-project`) so probably just `multibranch` is clear enough.

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java (hunk 1 above)
Comment (joseblas) [reply src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java]: you are right, got confused.

File: src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java (hunk 2 above)
Comment (joseblas) [reply src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java]: removed.

**Review by jglick — APPROVED** [review]
